eth/downloader/downloader_test.go

```diff
@@ -535,125 +534,6 @@ func testThrottling(t *testing.T, protocol uint, mode SyncMode) {
 	}
 }
 
-// Tests that simple synchronization against a forked chain works correctly. In
```

Comment:
These fork tests are no longer relevant: we no longer actively track multiple forks now that the chain progression is determined by the skeleton.